reV/supply_curve/point_summary.py

```diff
@@ -428,6 +379,153 @@ def mean_h5_dsets_data(self):
 
         return _mean_h5_dsets_data
 
+    @staticmethod
+    def _mode(data):
```

Comment:
This seems super unnecessary

Reply:
I agree its a bit silly, but I had to wrap it to enable this dictionary:
https://github.com/NREL/reV/blob/94e9bab80dc3e46d8d1620a9b9199c2e9e6eac0d/reV/supply_curve/point_summary.py#L446-L451